internal/command/meta_encryption.go

```diff
@@ -0,0 +1,64 @@
+package command
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/opentofu/opentofu/internal/configs"
+	"github.com/opentofu/opentofu/internal/encryption"
+	"github.com/opentofu/opentofu/internal/encryption/config"
+	"github.com/opentofu/opentofu/internal/encryption/keyprovider/static"
+	"github.com/opentofu/opentofu/internal/encryption/method/aesgcm"
+	"github.com/opentofu/opentofu/internal/encryption/registry/lockingencryptionregistry"
+	"github.com/opentofu/opentofu/internal/tfdiags"
+)
+
+func (m *Meta) Encryption() (encryption.Encryption, tfdiags.Diagnostics) {
+	path, err := os.Getwd()
+	if err != nil {
+		return nil, tfdiags.Diagnostics{}.Append(fmt.Errorf("Error getting pwd: %w", err))
+	}
+
+	return m.EncryptionFromPath(path)
+}
+
+func (m *Meta) EncryptionFromPath(path string) (encryption.Encryption, tfdiags.Diagnostics) {
+	// This is not ideal, but given how fragmented the command package is, loading the root module here is our best option
+	// See other meta commands like version check which do that same.
+	module, diags := m.loadSingleModule(path)
+	if diags.HasErrors() {
+		return nil, diags
+	}
+	enc, encDiags := m.EncryptionFromModule(module)
+	diags = diags.Append(encDiags)
+	return enc, diags
+}
+
+func (m *Meta) EncryptionFromModule(module *configs.Module) (encryption.Encryption, tfdiags.Diagnostics) {
+	reg := lockingencryptionregistry.New()
+	if err := reg.RegisterKeyProvider(static.New()); err != nil {
+		panic(err)
+	}
+	if err := reg.RegisterMethod(aesgcm.New()); err != nil {
+		panic(err)
+	}
+
+	cfg := module.Encryption
+	var diags tfdiags.Diagnostics
+
+	envName := "TF_ENCRYPTION"
```

Comment:
Make this a constant with a comment explaining what it does. Also please add this to the docs here: https://opentofu.org/docs/cli/config/environment-variables